docs/users/coordinator.md

```diff
@@ -0,0 +1,277 @@
+# Coordinator
+This tutorial focuses on how to launch a Hermez Coordinator node on localhost using the available tools. It includes two parts:
+1. [Launching a Coordinator Node](#coordinator-node)
+2. [Launching a Proof Server](#proof-server)
+
+## Coordinator Node
+### Dependencies
```

Comment:
I'd recommend to ask the coordinator to compile using Go 1.16 because it has the latest version of crypto adjustments that increase the security.